static/js/entry/dashboard.js

```diff
@@ -25,11 +28,30 @@ const store = configureStore();
 let debug = SETTINGS.reactGaDebug === "true";
 ga.initialize(SETTINGS.gaTrackingID, { debug: debug });
 
+const rootEl = document.getElementById("dashboard");
 ReactDOM.render(
-  makeDashboardRoutes(
-    browserHistory,
-    store,
-    () => ga.pageview(window.location.pathname)
-  ),
-  document.getElementById("dashboard")
+  <AppContainer>
```

Comment:
Yeah, you can read about how it works here: https://github.com/gaearon/redux-devtools/commit/64f58b7010a1b2a71ad16716eb37ac1031f93915#diff-363d10e8c7d704f50ac23a129f4a123cR11